docs/source/user/installing.rst

```diff
@@ -65,13 +65,16 @@ versions installed on the system) as the required components are bundled into
 the llvmlite wheel.
 
 To use CUDA with Numba installed by `pip`, you need to install the `CUDA SDK
-<https://developer.nvidia.com/cuda-downloads>`_ from NVIDIA.  Then you may need to
-set the following environment variables so Numba can locate the required
-libraries:
+<https://developer.nvidia.com/cuda-downloads>`_ from NVIDIA.  Please refer to
+:ref:`cudatoolkit-lookup` for details.
 
-* ``NUMBAPRO_CUDA_DRIVER`` - Path to the CUDA driver shared library file
-* ``NUMBAPRO_NVVM`` - Path to the CUDA libNVVM shared library file
-* ``NUMBAPRO_LIBDEVICE`` - Path to the CUDA libNVVM libdevice *directory* which contains .bc files
+.. note:: Previous versions suggest the following deprecated environment
+    variables to be set:
+
+    * ``NUMBAPRO_CUDA_DRIVER`` - Path to the CUDA driver shared library file
+    * ``NUMBAPRO_NVVM`` - Path to the CUDA libNVVM shared library file
+    * ``NUMBAPRO_LIBDEVICE`` - Path to the CUDA libNVVM libdevice *directory*
+      which contains .bc files
```

Comment:
Numba should now be able to detect CUDA libraries installed system-wide on Linux and Windows.

Reply:
Linux only.